x-pack/platform/plugins/shared/actions/server/lib/gen_ai_token_tracking.ts

```diff
@@ -330,11 +330,12 @@ export const getGenAiTokenTracking = async ({
   return null;
 };
 
-export const shouldTrackGenAiToken = (actionTypeId: string) =>
-  actionTypeId === '.gen-ai' ||
-  actionTypeId === '.bedrock' ||
-  actionTypeId === '.gemini' ||
-  actionTypeId === '.inference';
+export const shouldTrackGenAiToken = (actionTypeId: string, subAction?: string) =>
```

Comment:
https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/actions/server/lib/action_executor.ts#L593

is the only line where this function is called apart from its tests, and we do not pass `subAction` here, is this param needed?